src/main/java/fasttrack/logic/parser/ParserUtil.java

```diff
@@ -78,9 +78,7 @@ public static String parseExpenseName(String expenseName) throws ParseException
      */
     public static Price parsePrice(String price) throws ParseException {
         requireNonNull(price);
-        // remove trailing zeroes, commas, dollar signs, and spaces
-        String trimmedPrice = price.trim().replaceAll("0*$", "").replaceAll(",", "").replaceAll("\\$", "")
-                .replaceAll(" ", "");
+        String trimmedPrice = price.trim();
```

Comment:
Should we find another regex to remove the dollar signs, commas and spaces that doesn't end up removing trailing zeroes instead of just trimming?